sway-core/src/language/ty/declaration/trait.rs

```diff
@@ -30,6 +30,7 @@ pub struct TyTraitDecl {
     pub supertraits: Vec<parsed::Supertrait>,
     pub visibility: Visibility,
     pub attributes: transform::AttributesMap,
+    pub call_path: CallPath,
```

Comment:
This needs a better, more descriptive name, as it seems you want to guarantee that it'll be a full path to the trait decl and not just any `CallPath`.